task/bq2bq/executor/bumblebee/transformation.py

```diff
@@ -397,9 +403,8 @@ def transform(self):
             tasks.append(task)
             query_index += 1
 
-        # executing serially for now
-        for task in tasks:
-            task.execute()
+        executor = ConcurrentTaskExecutor(self.concurrency)
+        executor.execute(tasks)
```

Comment:
Are we okay if the order of these executions is not serial in concurrent runs? Although I can see default concurrency will be 1 but can you confirm if the concurrency is 1, the order of these executions is still the same?

Reply:
We had made a little change on the `ConcurrentTaskExecutor` which previously run in reversed order.

Now, its all good on our side, and has been tested that the execution run in correct order if the concurrency is 1